core/nuxeo-operation.js

```diff
@@ -330,27 +393,90 @@ import './nuxeo-connection.js';
         });
     }
 
+    /**
+     * Handler to abort bulk operations executed through the BAF
+     */
+    _abort(commandId) {
+      // CHECK - is this enough to abort the previous request?
+      if (this._controller) {
+        this._controller.abort();
+      }
+      // if the operation is aborted, then on next poll we get the correct state
+      return this.$.nx.request().then((request) =>
+        request
```

Comment:
This `request` being here causes an extra level of indentation on the following 20+ lines, but I'm assuming this is `prettier` at work.

Reply:
Indeed it is 😬